bin/packages/build.js

```diff
@@ -68,6 +73,27 @@ function buildFile( file, silent ) {
 	buildFileFor( file, silent, 'module' );
 }
 
+function buildStyle( packagePath ) {
+	const styleFile = path.resolve( packagePath, SRC_DIR, 'style.scss' );
+	const outputFile = path.resolve( packagePath, BUILD_DIR.style, 'style.css' );
+	mkdirp.sync( path.dirname( outputFile ) );
+	const builtSass = sass.renderSync( {
```

Comment:
I know it's convenient for development, but a build script responsible for generating many distinct modules seems particularly write for an _async_ implementation, not sync.

Reply:
yes, true. I think this could be handled separately as it has implicatons on the whole `build` script and not only the styles build script.